### PR TITLE
switch to python3

### PR DESCRIPTION
**Pull request by tchaikov**

### CODE REVIEW

**Comment by tchaikov** [comment]
tested using `src/script/run-cbt.sh` in ceph/ceph

**Comment by tchaikov** [comment]
being tested at http://pulpito.ceph.com/kchai-2019-12-20_05:00:25-rados:perf-master-distro-basic-smithi/

**Comment by liewegas** [comment]
will this work on el7 for nautilus?

**Comment by tchaikov** [comment]
@liewegas i think we can create a branch just for nautilus.

**Comment by markhpc** [comment]
@tchaikov I would very much like to avoid separate branches for el7 if we can help it.  I'll take a look.

File: README.md [hunk 1]
```diff
@@ -51,9 +51,9 @@ machine.
 
 CBT uses several libraries and tools to run:
 
- 1. python-yaml - A YAML library for python used for reading 
+ 1. python3-yaml - A YAML library for python used for reading
```
Comment (tchaikov) [inline README.md]: we might need to install `python36-PyYAML` and `python36-lxml` on centos7.

**Comment by tchaikov** [comment]
@liewegas @markhpc patch for nautilus branch posted at https://github.com/ceph/ceph/pull/32436

**Pull request merged**